<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuitarSetting</class>
 <widget class="QWidget" name="GuitarSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="strings_label">
       <property name="text">
        <string>Strings</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="strings_input"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="frets_label">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="text">
        <string>Frets</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="frets_input">
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="tuning_groupbox">
       <property name="title">
        <string>Tuning</string>
       </property>
       <widget class="QTextEdit" name="tuning_input">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
